docs/changelog/115399.yaml

```diff
@@ -10,9 +10,8 @@ breaking:
     In this release (8.16), we have introduced major changes to the retrievers framework and how they can be evaluated,
     focusing mainly on compound retrievers like `rrf` and `text_similarity_reranker`, which allowed us to support
     additional search features like collapsing, explaining, aggregations, and highlighting.
-    To ensure consistency and avoid unexpected failures, we have introduced a new cluster_feature that
-    all nodes must satisfy, in order for the `rrf` and `text_similarity_reranker` retriever queries to be evaluated.
-    This guarantees that all nodes are able to take advantage of the new framework and provide compatible responses.
+    To ensure consistency and avoid unexpected failures, `rrf` and `text_similarity_reranker` retriever queries are no longer compatible
+    with previous releases ( <= 8.15).
```

Comment:
YIKES, this implies that `rrf` and `text_similarity_reranker` no longer works! Please make sure you are very clear that they only break in a mixed cluster scenario, not that they stop working in 8.16 :)

Reply:
hahah that's definitely not what I was going for 😅 had the mixed cluster description in the `impact` section below,
so I thought to not include it here as well. 

WDYT about the following? 
```
In this release (8.16), we have introduced major changes to the retrievers framework 
and how they can be evaluated, focusing mainly on compound retrievers 
like `rrf` and `text_similarity_reranker`, which allowed us to support full 
composability (i.e. any retriever can be nested under any compound retriever), 
as well as supporting additional search features like collapsing, explaining, 
aggregations, and highlighting.

To ensure consistency, and given that this rework is not available until 8.16, 
`rrf` and `text_similarity_reranker`  retriever queries would now  
throw an exception in a mixed cluster scenario, where there are nodes 
both in current or later (i.e. >= 8.16) and previous ( <= 8.15) versions.

As part of the rework, we have also removed the `_rank` property from 
the responses of an `rrf` retriever.
```